### DISCUSSION TITLE
Error: Argument is not numeric nor factor but Date

### DISCUSSION THREAD

**Discussion by GChiarenza** [easystats/insight#808] [General]
My model is a mixed regression with "scale(Date)" as a fixed effect. Date is the day my experiment took place and is converted using as.Date(rounds$date, "%d-%b-%y"). I am using estimate_mean() and estimate_contrasts() but it seems they don't like dates? is tehre a turnoarund or should i convert back date as a numeric variable? Thanks :)

**Comment by bwiernik** [comment]
Rather than scaling a Date variable, I would recommend centering the variable around the start date and then converting it to numeric so it is "days since start". That will help with support functions like in easystats and produce results that are easier to interpret.
